sympy/matrices/matrices.py

```diff
@@ -4371,8 +4369,13 @@ def norm(self, ord=None):
         # Matrix Norms
         else:
             if ord == 1:  # Maximum column sum
-                m = numpy.absolute(self)  # m is a numpy array with absolute value of the original matrix
-                return numpy.sum(m.max(1))  # returns the sum of the maximum column
+                column = [] # keeps track of the number of columns
+
+                for i in range(self.shape[1]) :
+                    column.append(i)
```

Comment:
Oh Yes. How silly of me not to notice that. Making a list from a list. Redundant steps. I will fix it.